code/mergeblock/merge.py

```diff
@@ -0,0 +1,86 @@
+#Jakub Podolak for Plastic Monkeys Cansat 2019
+
+import subprocess
+import configparser
+import msgpack
+import time
+import sys
+import re
+
+#function to read paths to sensors' logs from config file
+def get_directories(config):
+    directories = []
+    paths = config.items("PATHS")
+    for key, path in paths:
+        directories.append(path)
+    return directories
+
+#function reads sensors' names from config file
+def get_keys(config):
+    keys = []
+    paths = config.items("PATHS")
+    for key, path in paths:
+        keys.append(key)
+    return keys
+
+#function takes paths to logs as an argument, returns list of the latest readings(int) from all paths
+def get_tails(directories,length):
+    tails = []
+    for x in directories:
+        with open(x, 'rb') as fh:
+            fh.seek(-1*length, 2)
+            last = fh.readlines()[-1].rstrip().decode()
+        all_numbers = re.findall('\d+',last)
```

Comment:
it will work only for integers, won't work for floating point numbers, are we okey with that?

Reply:
Supposing that we are converting everything to integers it is okay

Reply:
We discussed it - we are using only integers so it will work fine